substrate/frame/identity/src/benchmarking.rs

```diff
@@ -929,5 +941,52 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn migration_v3_username_step() -> Result<(), BenchmarkError> {
```

Comment:
we could have those benchmarks in a different benchmark set. We can decouple benchmarks from pallets a bit by having some dedicated pallet only implemented for benchmarking like `frame-session-benchmarking`.
In the long term those inlined benchmarks will have to be removed. Maybe it is better to have the migration in a dedicated sub-module, not sure.